Header header
sensor_msgs/Image Camera
sensor_msgs/CameraInfo CameraInfo
